src/main/java/com/alkemy/ong/mapper/SlideMapper.java

```diff
@@ -32,4 +33,14 @@ public static SlideDTO mapDomainToDto(Slide slideDomain) {
         return slideDTO;
     }
 
+    public static Slide mapCreationDTOToDomain(SlideCreationDTO slideCreationDTO) {
+        Slide slideDomain = Slide.builder()
+                .image(slideCreationDTO.getImage())
+                .text(slideCreationDTO.getText())
+                .organizationOrder(slideCreationDTO.getOrder())
+                .id(slideCreationDTO.getOrganizationId())
```

Comment:
Este campo es el id del slide. Hay que setearlo a organization:
```suggestion
                .organization(Organization.builder().id(slideCreationDTO.getOrganizationId()).build())
```